### PR TITLE
Enable ruling for non-css files

### PR DESCRIPTION
**Pull request by andrea-guarino-sonarsource**

### CODE REVIEW

File: its/ruling/src/test/java/org/sonar/css/its/CssRulingTest.java [hunk 1]
```diff
@@ -43,22 +45,50 @@
   @ClassRule
   public static final Orchestrator ORCHESTRATOR = Orchestrator.builderEnv()
     .setSonarVersion(System.getProperty(SQ_VERSION_PROPERTY, DEFAULT_SQ_VERSION))
+    .addPlugin(MavenLocation.of("org.sonarsource.php", "sonar-php-plugin", "LATEST_RELEASE"))
+    .addPlugin(MavenLocation.of("org.sonarsource.html", "sonar-html-plugin", "LATEST_RELEASE"))
+    .addPlugin(MavenLocation.of("org.sonarsource.javascript", "sonar-javascript-plugin", "LATEST_RELEASE"))
+    .addPlugin(MavenLocation.of("org.sonarsource.typescript", "sonar-typescript-plugin", "LATEST_RELEASE"))
```
Comment (saberduck) [inline its/ruling/src/test/java/org/sonar/css/its/CssRulingTest.java:51]: why we need typescript plugin? It shouldn't be needed, we analyze TS with sonarjs

**Review by saberduck — CHANGES_REQUESTED** [review]
we shouldn't need typescript plugin

File: its/ruling/src/test/java/org/sonar/css/its/CssRulingTest.java (hunk 1 above)
Comment (andrea-guarino-sonarsource) [reply its/ruling/src/test/java/org/sonar/css/its/CssRulingTest.java:51]: Indeed, I copy pasted from https://github.com/SonarSource/sonar-css/blob/0ca087fa3e2fd45f28897194ee7380ca225e4655/its/plugin/src/test/java/org/sonar/css/its/Tests.java#L63 

and I forgot to remove sonar-typescript.

Thanks!

**Review by saberduck — APPROVED** [review]

File: its/ruling/src/test/java/org/sonar/css/its/CssRulingTest.java (hunk 1 above)
Comment (saberduck) [reply its/ruling/src/test/java/org/sonar/css/its/CssRulingTest.java]: can you also remove it from plugin test? it shouldn't be there

**Pull request merged**